### PR TITLE
Elastic collision handler with maven

### PR DESCRIPTION
**Pull request by Oskmey**
Maven setup with elastic collision.

### CODE REVIEW

File: crocket/src/test/java/com/crocket/AppTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+{
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue()
```
Comment (filipkauffeldt) [inline crocket/src/test/java/com/crocket/AppTest.java:16]: Could be removed

File: crocket/src/main/java/com/crocket/ElasticCollision.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.crocket;
+import java.lang.Math;
+
+public class ElasticCollision {
+    public static void calculateElasticCollision(Entity entity1, Entity entity2){
```
Comment (filipkauffeldt) [inline crocket/src/main/java/com/crocket/ElasticCollision.java:5]: Could be renamed to elasticCollision

File: crocket/src/main/java/com/crocket/ElasticCollision.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package com.crocket;
+import java.lang.Math;
+
+public class ElasticCollision {
```
Comment (filipkauffeldt) [inline crocket/src/main/java/com/crocket/ElasticCollision.java:4]: Could maybe be a more general CollisionHandler

**Review by filipkauffeldt — APPROVED** [review]

**Review by Pontus-Wikstrom — APPROVED** [review]
looks good

**Pull request merged**